images/macos/helpers/Common.Helpers.psm1

```diff
@@ -36,6 +36,7 @@ function Get-OSVersion {
         IsMonterey = $osVersion.Version.Major -eq "12"
         IsVentura = $($osVersion.Version.Major -eq "13" -and $processorArchitecture -ne "arm64")
         IsVenturaArm64 = $($osVersion.Version.Major -eq "13" -and $processorArchitecture -eq "arm64")
+        IsSonoma       = $($osVersion.Version.Major -eq "14")
```

Comment:
How about to check here architecture? Since we already know what changes will follow this.

Reply:
No, I think it’s irrelevant until there’s a reason to check it, but for now there isn’t one.

Reply:
Well, it's a good point, but I'm already proceed with building arm image and going to update this part in the next PR.